include/fastdds/dds/topic/ContentFilteredTopic.hpp

```diff
@@ -49,9 +49,7 @@ class ContentFilteredTopic : public TopicDescription
 
     RTPS_DllAPI ContentFilteredTopic(
             const std::string& name,
-            Topic* related_topic,
-            const std::string& filter_expression,
-            const std::vector<std::string>& expression_parameters);
+            Topic* related_topic);
```

Comment:
The constructor is private but I am not sure if the virtual table would be modified by this change and thus affect ABI compatibility in `2.5.x`. Nobody is surely using this API because it was not previously implemented, but we should be aware of this issue.

Reply:
@EduPonz What do you think about this comment?

Reply:
Kept ABI compatibility on commit 323fb2fcc9120da3bd0468ab931f1e1b4458d4ed